internal/pkg/pkg.go

```diff
@@ -695,3 +699,49 @@ func SetPkgPathAnnotation(rn *yaml.RNode, pkgPath types.UniquePath) error {
 func RemovePkgPathAnnotation(rn *yaml.RNode) error {
 	return rn.PipeE(yaml.ClearAnnotation(pkgPathAnnotation))
 }
+
+// RGFile returns the resourcegroup object by lazy loading it from the filesytem.
+func (p *Pkg) RGFile(filename string) (*rgfilev1alpha1.ResourceGroup, error) {
```

Comment:
Suggest naming it as a operation (e.g. verb + tasks) like ReadRGFile or GetRGFile.

Reply:
Done.